spring-boot-actuator/src/main/java/org/springframework/boot/actuate/health/AbstractHealthIndicator.java

```diff
@@ -31,13 +34,19 @@
  */
 public abstract class AbstractHealthIndicator implements HealthIndicator {
 
+	private static final Log logger = LogFactory.getLog(AbstractHealthIndicator.class);
+
 	@Override
 	public final Health health() {
 		Health.Builder builder = new Health.Builder();
 		try {
 			doHealthCheck(builder);
 		}
 		catch (Exception ex) {
+			logger.warn(String.format(
+					"%s failed: %s",
+					this.getClass().getSimpleName(),
```

Comment:
Wouldn't it be better to have a non static logger and use `getClass()` as the source? That way the classname of the health indicator would show up in the log entry by default.